kdl_parser/include/kdl_parser/visibility_control.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
year

Reply:
Actually, for that matter, this should be the BSD license to match the rest of the code anyway.  I'll fix both of those.